openlcs/packages/mixins.py

```diff
@@ -210,3 +254,93 @@ def save_scan_result(self, **kwargs):
                     self.save_file_copyright_scan(list(set(file_ids)))
                     self.save_copyright_detections(path_file_dict, data)
                     self.update_scan_flag(source, "copyright_scan")
+
+
+class SaveContainerComponentsMixin:
+    def __init__(self):
+        self.components = None
+        self.release = None
+
+    @staticmethod
+    def create_component(component_data):
+        component, _ = Component.objects.update_or_create(
```

Comment:
while creating components, need to tell whether the `uuid` is from corgi(synced=True) or not(synced=False). The `corgi_handler` module doesn't tell this, thus how do you set `synced` based on `component_data`?

Reply:
updated， add `synced`